src/css/bundle.scss

```diff
@@ -497,10 +532,29 @@ input[type='text'].display {
 
         >.item {
             // Make tabs stretch across horizontally.
+            color: white;
             flex: 1;
             text-align: center;
         }
     }
+    .sheet .editor-content {
+        color: white;
+        font-family: 'klavika-medium';
```

Comment:
Check _variables.scss for $systemFonts and add a new one $editorFonts = 'klavika-medium'. Again, to keep it consistent. :)